app/src/main/java/com/example/cactusnotes/log_in/LogInActivity.kt

```diff
@@ -17,13 +19,13 @@ class LogInActivity : AppCompatActivity() {
         setContentView(binding.root)
 
         binding.buttonLogIn.setOnClickListener {
-            validate(binding.emailorUsernameTextInputLayout)
-            validate(binding.passwordLogInTextInputLayout)
+            isEmailOrUsername()
+            isPassword()
```

Comment:
The function names does not clearly tells about what they do. A better naming could be:

validateEmailOrUsername()
validatePassword()